proposals/kubeflow-versioning.md

```diff
@@ -0,0 +1,54 @@
+# Kubeflow Versioning
+
+## Motivation
+
+Regularize the Kubeflow release schedule.
+
+## Proposal
+
+In brief, this proposal would switch the release cycle to this:
+
+![Kubeflow Versioning](diagrams/kubeflow-versioning.png)
+
+The changes this would introduce are:
+
+- A new version of Kubeflow would be released every 2 months
+- The deadline would be strict; if something is not ready for a particular release, it will be bumped to the next one
+- Each version would be supported (i.e. bugs/security issues fixed) for 4 months, except for for yearly Extended Support (ES) versions that would be supported for 16 months
```

Comment:
`for` repeated

Reply:
I am not sure the community has a concept of `supported` well defined yet but I think it makes sense to add a flag of `supportable`, meaning that organizations that offer Kubeflow support can have that as a reference.